### ISSUE TITLE
Not all Info.plist files have a CFBundleName (thread 'main' panicked at 'IndexMap: key not found', src\bundle.rs:50:27)

### ISSUE THREAD

**Issue by ghost** [hikari-no-yume/touchHLE#10]
```
PS D:\다운로드\touchHLE_v0.1.0_Windows_x86_64> .\touchHLE.exe "OvenBreak.app"
touchHLE v0.1.0 — https://touchhle.org/

thread 'main' panicked at 'IndexMap: key not found', src\bundle.rs:50:27
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
This happen when I try to run Ovenbreak (paid version) 1.09 ~ 1.12 ipa.

```
PS D:\다운로드\touchHLE_v0.1.0_Windows_x86_64> .\touchHLE.exe "OvenBreak.app"
touchHLE v0.1.0 — https://touchhle.org/

Error: "Could not parse icon image"
```
As an aside, this error happen when I try to run Ovenbreak (paid version) 1.0 ~ 1.08 ipa, but seems like it already commented on other thread.

**Comment by hikari-no-yume** [comment]
Oh right, some app bundles don't have the CFBundleName key in their Info.plist file, I should fix that eventually.

To be clear, exactly one app (Super Monkey Ball) is expected to work right now, but I'll keep this open as a reminder of this particular issue. It's extremely likely that there are dozens or hundreds of other issues that would prevent OvenBreak from working.

**Comment by hikari-no-yume** [comment]
I fixed this recently in c1c746202b499f1b1c9c3b2acd04bf7de7968637 but forgot about this issue.
